docs/src/documentation/03-components/01-action/button/03-accessibility.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/button/accessibility/
+---
+
+# Accessibility
+
+## Button
+
+The Button component has been designed with accessibility in mind.
+
+The following props provide additional information to screen readers:
+
+- Use `ariaControls` prop to add `aria-controls` attribute to establish the relationship between button and element which is controlled by it. `aria-controls` works only with a unique `id` of an element.
+
+- Use `ariaExpands` prop to add `aria-expands` to indicate screenreaders, that element controlled by button through `ariaControls` is expanded or not.
+
+- Use `disabled` prop to indicate users that button is inactive and they can't interact with it.
```

Comment:
If we are mentioning this prop here, we should tell what happens on assistive technologies when the button is disabled. Is it announced? Can we focus it? Or is it completely ignored?